path_role.go

```diff
@@ -49,6 +49,12 @@ are allowed.`,
 					Type: framework.TypeCommaStringSlice,
 					Description: `List of namespaces allowed to access this role. If set to "*" all namespaces
 are allowed.`,
+				},
+				"bound_service_account_namespace_selector": {
+					Type: framework.TypeString,
+					Description: `A label selector for Kubernetes namespaces which are allowed to access this role.
+Accepts either a JSON or YAML object. If set with bound_service_account_namespaces,
+the conditions are ORed.`,
```

Comment:
I think the answer to the additional questions is it's consistent with the k8s secret engine. In retrospect, using OR rather than AND does seem a bit surprising, but I think we should probably maintain consistency for now.

Reply:
> I think the answer to the additional questions is it's consistent with the k8s secret engine. In retrospect, using OR rather than AND does seem a bit surprising, but I think we should probably maintain consistency for now.

@tomhjp  Ah, so you are referring to https://developer.hashicorp.com/vault/api-docs/secret/kubernetes#allowed_kubernetes_namespace_selector? I guess that will make the UX a bit more consistent.